### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Default Documents_Keyboard: Focus is not visible on the entire page.

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#111]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator,NVDA
**MAS ID Violated:** MAS2.4.7

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Default Documents and press enter
6) Verify that bold frame(dotted or plain) is there to distinguish between the focused elements on the page.

**Actual Result:**
Focus is not visible on the entire page

**Note:**
Same issue is observed through out the application

**Expected Result:**
Focus should be visible clearly on all the controls on page. Its impossible to identify user is on which control

**User Impact:** 
Visually Impaired Users will face difficulty in navigating through the contents and controls if focus is not visible properly 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachments:**
![bug_111_focus visible_issue](https://user-images.githubusercontent.com/42638915/44661880-ae553180-aa29-11e8-8fe4-256e0a20a748.jpg)

![bug_111_focus is not visible on some controls of rewrite maps page](https://user-images.githubusercontent.com/42638915/44908027-a3efac00-ad37-11e8-985c-25cee1c9adc1.jpg)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.7; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
